internal/seclang/parser_test.go

```diff
@@ -169,6 +169,7 @@ func TestHardcodedIncludeDirectiveDDOS2(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	tmpFile2.Close()
```

Comment:
shouldn't we assert errors on tmpFile2.Close()?

Reply:
Done